### ISSUE TITLE
Deploy "client/" on a host with no server

### ISSUE THREAD

**Issue by fplgusmao** [johnpapa/generator-hottowel#131]
I'm using your template to make my AngularJS app. I need to deploy it on my college machines, so I think I can't use any of your `server/` code because I have no permissions for having a `node.js` server running and waiting for requests...

Previously, I wasn't following this template and all I needed to do was to upload the files to the host. Then, `index.html` would bring all the stuff together, and the data-services would fetch `json` files directly, using `$http.get`, instead of creating any server app or script.

From what I understood, for what I need and using this template, I just need to grab the `build/` and upload it to the host? I am trying to do that, but it gives me an error, stating that it can't access the `JS` and `CSS` files:

`Failed to load resource: the server responded with a status of 404 (Not Found)`

Is it something I am doing wrong or is it server-related?

**UPDATE:** The 404, as expected, is complaining about not finding the files. When I looked into its details, I found that the `index.html` is trying to fetch from the wrong path. In my case, it should be fetching from, say:

`http://web.school.com/my-school-id/styles/lib-c34ac1cee9.css`

But instead it's trying to access this link:

`http://web.school.com/styles/lib-c34ac1cee9.css`

So, it seems like it's eating up part of the link, but I don't understand why, since I didn't change any of the file paths, nor the `<base href="...">`

**Comment by fplgusmao** [comment]
I think I solved the problem. But the solution is not that pretty...

I changed the `index.html` `<base>` to `href="./"` trying to make it a relative path. This solved the 404's, but I'm still doubtful about the validation of both this solution and the overall method of just uploading the `build/` folder to the server. Is this really what I should be doing?

Sorry for the noobiness

**Comment by fplgusmao** [comment]
My previous solution solved the 404's but a problem with the `ui-router` emerged, by doing that. So, instead of changing the `base href` I changed the dependencies path. The best way I found to do this was changing the `index.html` comments, like:

 `<!-- build:css styles/lib.css -->`

to:

`<!-- build:css my-school-id/styles/lib.css -->`

This way, the `gulp-useref` would inject the dependencies on the final `index.html` file, with the current path. The same plugin also creates a "my-school-id" folder inside the build folder, as supposed. I had to take its contents and place them on the root. Then, finally, I would upload the build folder, expect for the "my-school-id", to the server. **It finally worked!**

**PS:** this problem probably won't happen to most people, since I now understand that it was related to how my school server was structured. The dependency paths would start right away from the server route, and to most people, this wouldn't matter. In my case, however, I still needed to pass through my student directory.
